stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/composer/InputField.kt

```diff
@@ -56,6 +56,24 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  * @param innerPadding The padding inside the input field, around the label or input.
  * @param decorationBox Composable function that represents the input field decoration as it's filled with content.
  */
+@Deprecated(
+    message = "Deprecated in favor of new InputField implementation providing more padding customization options.",
+    replaceWith = ReplaceWith(
+        expression =
+        "public fun InputField(" +
+            "value: String," +
+            "onValueChange: (String) -> Unit," +
+            "modifier: Modifier = Modifier," +
+            "enabled: Boolean = true," +
+            "maxLines: Int = Int.MAX_VALUE," +
+            "border: BorderStroke = BorderStroke(1.dp, ChatTheme.colors.borders)," +
+            "horizontalPadding: Dp = 18.dp," +
+            "verticalPadding: Dp = 8.dp," +
+            "decorationBox: @Composable (innerTextField: @Composable () -> Unit) -> Unit)",
+        imports = ["io.getstream.chat.android.compose.ui.components.composer"]
+    ),
```

Comment:
To be honest I have never understood how this is supposed to work. Doesn't work for me when autocorrecting the warning in Android Studio. Could you check if the following snippet: 

```
@Deprecated(
    message = "Deprecated in favor of new InputField implementation providing more padding customization options.",
    replaceWith = ReplaceWith(
        expression = "InputField(value, onValueChange, modifier, enabled, maxLines, border, horizontalPadding, verticalPadding, decorationBox)",
        imports = ["io.getstream.chat.android.compose.ui.components.composer"]
    ),
    level = DeprecationLevel.WARNING
)
```
<img width="930" alt="Screenshot 2022-05-25 at 18 04 34" src="https://user-images.githubusercontent.com/9600921/170294988-ea7eeb42-d7e9-4f32-a23b-0eac5d759b7e.png">

Reply:
Yes it works like this but is pretty broken and will add all the missing parameters not passed in. I honestly don't think this change is worth it, but I can add it.

Reply:
Do we really need to deprecate the old input field? We've just exposed one new parameter that _has a default value_ hence it's not going to be breaking for any of our users, they can just expand on their behavior.